kickstarter/android-oss
# WEB-846: [Android] Bring in localized strings for consent management

# 📲 What

Strings for the consent management modal have been translated, so we needed to update with the new copy. 

# 👀 See

![Screenshot_20230221-220220](https://user-images.githubusercontent.com/19390326/220469483-552da1de-6a17-4211-a100-c59b4daf325c.png)

# 📋 QA

- Delete all app data and cache if you have the app installed already
- Make sure you are on staging
- Modal should appear on app launch with the correct copy

# Story 📖

[WEB-846: [Android] Bring in localized strings for consent management](https://kickstarter.atlassian.net/browse/WEB-981)
